lib/peep.rb

```diff
@@ -0,0 +1,39 @@
+require "pg"
+
+class Peep 
+	attr_reader :message, :date
+	def initialize(message, date = '')
+		@message = message
+		@date = date
+	end
+	def self.all 
+		if (ENV["ENVIRONMENT"] == 'test')
+			connection = PG.connect(dbname: 'chitter_test')
+		else 
+			connection = PG.connect(dbname: 'chitter')
+		end
+		result = connection.exec("SELECT * FROM peeps ORDER BY id DESC;")
```

Comment:
Nice! I can see that this would work. Since we're ordering by time it would be more explicit for the query to order by the date field. This could work by making the field type in postgres a timestamp rather than a string.